### PR TITLE
[Docbug][ESlintsupport][3879]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added ESLint support question to Manifest file. Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0d90eb25f1b4a1aad93ede3d4677ff8a43cd019c/msteams-platform/tabs/how-to/create-personal-tab.md) | 98 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b3ba3c9f-cb7b-4e61-892a-f10c19084088) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/how-to/create-personal-tab.md [hunk 1]
```diff
@@ -87,19 +87,19 @@ In addition, this project requires that you have the following installed in your
 
     **Quick scaffolding? (Y/n)**
 
-    The default is yes; enter **n** to enter your Microsoft Partner Id.
+    The default is yes; enter **n** to enter your Microsoft Partner ID.
 
     **Enter your Microsoft Partner Id, if you have one? (Leave blank to skip)**
 
-    This field is not required and should only be used if you are already part of the [Microsoft Partner Network](https://partner.microsoft.com).
+    This field isn't required and must be only used if you're already part of the [Microsoft Partner Network](https://partner.microsoft.com).
```
Comment (v-abirade) [inline msteams-platform/tabs/how-to/create-personal-tab.md:94]: must be used only if you're already part of the...

**Review by v-abirade — CHANGES_REQUESTED** [review]
@v-rodriguezd,

Please make the changes.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b02c983e1b1f89de3304fab007105c20095570c2/msteams-platform/tabs/how-to/create-personal-tab.md) | 98 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1fc94d5-3b3f-4860-89ac-934e9444d3d4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/79ee2b818389594f8ebc4020dae3841f3d961f66/msteams-platform/tabs/how-to/create-personal-tab.md) | 98 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0bb4cc66-35eb-461f-a99d-933997f5875a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approve

**Pull request merged**